app/models.py

```diff
@@ -30,11 +30,12 @@ class Rides:
     """
 
     def __init__(self):
-        pass
+        self.all_rides = {}
```

Comment:
i dont know your case but i think all_rides dictionary should be a class variable i.e move it out of the __init__.py